docs/source/release/v6.13.0/Diffraction/Engineering/Bugfixes/38882.rst

```diff
@@ -0,0 +1 @@
+- `#38882 <https://github.com/mantidproject/mantid/issues/38882>`_ : Fix issue with ``GSAS-II GSASIIscriptable.py`` hard-coded path which is invalid for newer version of GSAS-II.
```

Comment:
Probably worth adding a version number which we know doesn't work on main?

Reply:
Good point! There may be other older versions of GSAS-II between the one installed on IDAaaS (`4579`) and the most recent version I have tested with (`5758`) that may also cause a path issue prior to this PR that I'm not aware of, however we should specify at least one known version that breaks on main currently. I will update the release notes to include the newest version of GSAS-II `5758` that this PR aims to resolve.

Reply:
Yep I think there's no point going through GSAS github trying to find out when the directory changes, but agreed, if you could add that it will now work for version `5758` as you suggest - that would be great thanks!